docs/cookbook/html-conversion.md

```diff
@@ -0,0 +1,980 @@
+
+Finally, delete all of the markup in the `beforeMain` and `afterMain` sections. into the `beforeMain` block render the navigation - `{% render navigation.navigationArea() %}`. In the `afterMain` block render the footer fragment = `{% render footer.footerArea() %}`. The final modification we can make to the template is to add the semantic `<main></main>` tags. Looking at the original template pages, we can see that the main section has a class of `mb-4`. This markup can be added around the `{% block main %}{% endblock %}` tags.
+
+<AposCodeBlock>
+::: v-pre
+
+```django
+{% import 'fragments/navigation.html' as navigation %}
+{% import 'fragments/footer.html' as footer %}
+
+{# Automatically extends the right outer layout and also handles AJAX siutations #}
+{% extends data.outerLayout %}
+
+{% set title = data.piece.title or data.page.title %}
+{% block title %}
+  {{ title }}
+  {% if not title %}
+    {{ apos.log('Looks like you forgot to override the title block in a template that does not have access to an Apostrophe page or piece.') }}
+  {% endif %}
+{% endblock %}
+
+{% block extraHead %}
+  <!-- Font Awesome icons (free version)-->
+  <script src="https://use.fontawesome.com/releases/v6.1.0/js/all.js" crossorigin="anonymous"></script>
+  <!-- Google fonts-->
+  <link href="https://fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic" rel="stylesheet" type="text/css" />
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800" rel="stylesheet" type="text/css" />
+{% endblock %}
+
+{% block beforeMain %}
+  {% render navigation.navigationArea() %}
+{% endblock %}
+
+<main class="mb-4">
+{% block main %}
+  {#
+    Usually, your page templates in the @apostrophecms/pages module will override
+    this block. It is safe to assume this is where your page-specific content
+    should go.
+  #}
+{% endblock %}
+</main>
+
+{% block afterMain %}
+  {% render footer.footerArea() %}
+{% endblock %}
+
+```
+:::
+<template v-slot:caption>
+views/layout.html
+</template>
+</AposCodeBlock>
+
+Note the use of `navigationArea()` and `footerArea()` in the `render` calls. If you look at the fragment files, you will see that the fragment blocks use these names. This is to allow for a single fragment file to have multiple fragments. We are using these names just to be a little clearer in our calls, but they could just be named `navigation()` and `footer()`.
```

Comment:
> We are using these names just to be a little clearer in our calls, but they could just be named `navigation()` and `footer()`.

Probably just me, but I usually name all my fragments consistently so you don't have to remember the name of the render fn from the fragment. Here could just be `area()` so after you do your imports you'd have `navigation.area()` and `footer.area()`. Also, reads more naturally that way. Maybe could just be a note somewhere here since that change might make your fragment examples above a little confusing to a newbie.

Reply:
Changed the last sentence to reflect how you do this.